avmmakefile

```diff
@@ -49,7 +49,7 @@ checkovplancheck:
 
 fmtcheck: gofmtcheck tffmtcheck terrafmtcheck
 
-pr-check: fmtcheck tfvalidatecheck lint unit-test
+pr-check: clean fmtcheck tfvalidatecheck lint unit-test clean2
```

Comment:
Won't this impact people developing and testing locally? Deleting their state file may not be expected. I guess people should not be running this locally though.

Reply:
Yeah true. Though the pr-check needs to have a clean base to check that it works ok.